src/components/image-viewer/ImageInfo.vue

```diff
@@ -86,4 +82,9 @@
         background: rgba(0,0,0,0.7);
         padding:1rem;
     }
+
+    /* FIXME seems not to work */
+    icon-creative-commons, fa-icon {
```

Comment:
Is it the correct way to add styles to elements?

Reply:
It's better to add a dedicated class on your template. Do not bother about names collision, the `scoped` attribute on style part make class declared inside this block available only inside the component.

The only thing to take care of is to choose a class that is not used globally, e.g. bulma class, and class declared in [helper.scss](https://github.com/c2corg/c2c_ui/blob/master/src/assets/sass/helpers.scss)